docs/src/modules/components/DemoToolbar.js

```diff
@@ -437,6 +438,28 @@ export default function DemoToolbar(props) {
   return (
     <React.Fragment>
       <Root aria-label={t('demoToolbarLabel')} {...toolbarProps}>
+        {process.env.NEXT_PUBLIC_MUI_CHAT_API_BASE_URL && (
+          <OpenMuiChat
+            data-ga-event-category="mui-chat"
+            data-ga-event-label={demo.gaLabel}
+            data-ga-event-action="open-in-mui-chat"
+            {...getControlProps(10)}
```

Comment:
Something strange is happing here, this button is not accessible with the keyboard like all the other controls on the toolbar.